recipes/gunicorn/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "gunicorn" %}
+{% set version = "19.6.0" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version}}.tar.gz
+  url: https://github.com/benoitc/gunicorn/archive/{{ version }}.tar.gz
+  sha256: 1e0de4957bea60bfcff5215664bdfc3cf02c78e2aae9586766a9b4b437aebbb0
+
+build:
+  number: {{ build }}
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  preserve_egg_dir: yes
+  entry_points:
```

Comment:
original discussion: https://github.com/benoitc/gunicorn/issues/860 It looks like the pip patch linked to there just silences byte compile errors as long as the install itself succeeds. 

It doesn't sound like gunicorn folks have (or should) any desire to be "more" compatible.